contrib/debian/control.in

```diff
@@ -34,8 +34,7 @@ Depends: ${misc:Depends},
          shared-mime-info
 Recommends: python3,
 	    bolt,
-	    tpm2-tools,
-	    tpm2-abrmd,
```

Comment:
If i'm not mistaken the TSS library still will use tpm2-abrmd if installed right?  So probably want to keep tpm2-abrmd as a Recommends at least.

Reply:
It does use tpm2-abrmd if installed, yes. When using the Enhanced System API (ESAPI) like we do in this PR, I don't think tpm2-abrmd is really necessary because it will default to using the in-kernel resource manager (`/dev/tpmrm0`) available since kernel 4.12 anyway. Since tpm2-abrmd is generally useful for more complex application scenarios, I don't mind keeping it though, it is only in "recommended" anyway.

Reply:
Yeah, non-exclusive TPM access shouldn't be a problem with recent kernels. Come to think of it, there is another possible benefit of tpm2-abrmd though: if it is installed, ordinary users should be able to access the TPM out of the box without needing super user rights. The in-kernel resource manager OTOH is normally owned by the `tss` group, so users need to be manually added to this group to gain access. This might not be a concern for fwupd because it is probably run as root anyway.

Reply:
OK, I dropped tpm2-abrmd for now, let me know if it needs to be added back.